server/src/com/treode/server/SchematicStore.scala

```diff
@@ -22,6 +26,43 @@ class SchematicStore (store: Store, schema: Schema) {
     store.write (tx, ct, ops:_*)
   }
 
+  def batch_write(tx: TxId, ct: TxClock, req: Request): Async [TxClock] = {
+    val mapper = new ObjectMapper
```

Comment:
This creates a mapper per request, which is an unnecessary expense, because mappers can be used many times. You could just move this to an instance variable of the SchematicStore, but there's an even better option: use `textJson` that's already available from the package object.